docs/workflow/blocks/mapping.rst

```diff
@@ -54,15 +54,19 @@ Handy JMESPath patterns
 -----------------------
 
 Using a string as a value
-^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------
+
+
 To be valid JMESPath, keys and values must be wrapped in double quotes.
-For a hardcoded string, the value must be wrapped in backticks, otherwise it will result `null` for undefined variables.
+Hardcoded string's value must be wrapped in backticks, otherwise it will result `null` for undefined variables.
```

Comment:
Minor syntax changes - 

"A hardcoded string's value must be wrapped in backticks, otherwise it will result in `null` for undefined variables."